packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -139,6 +139,33 @@ depth = model.infer_image(raw_img) # HxW raw depth map in numpy
 	];
 };
 
+export const depth_pro = (model: ModelData): string[] => {
+	const installSnippet = `# Download checkpoint
+pip install huggingface-hub
+huggingface-cli download --local-dir checkpoints ${model.id}`;
+
+	const inferenceSnippet = `from PIL import Image
```

Comment:
I might be missing it but where are we using `Image` in the snippet?

Reply:
You are right, I'll update!